### DISCUSSION TITLE
Raspberry Pi 3B+

### DISCUSSION THREAD

**Discussion by jadonbrownza** [mikopbx/Core#164] [General ( Обо всём )]
After using imageusb to make bootable USB and SD Card, neither of them boots when starting the Raspberry Pi 3B+.

I believe it has to do with linux 4.x?

In the meantime I got it working using a virtual machine but would still like to get it running on Raspberry Pi 3B+.

Do you know why I cannot boot from bootable USB built using the release ISO?
If not, is there a way I can help you figure out what is causing the problem?

**Comment by jorikfon** [comment]
There is no way to start MikoPBX on the ARM platform. 
Now we are working on a Docker container. 
Maybe afterwards we will try to compile it to something like Raspberry.

**Reply by jadonbrownza** [reply]
Thank you Nikolay, I now understand why it would not work on Raspberry Pi.
I couldn't find anyone else trying it on Pi, but probably because all content I can find on the topic is in Russian 😄
